src/libtiled/tilelayer.h

```diff
@@ -513,6 +515,7 @@ class TILEDSHARED_EXPORT TileLayer : public Layer
     int mHeight;
     Cell mEmptyCell;
     QHash<QPoint, Chunk> mChunks;
+    QRegion mBounds;
```

Comment:
Since we only care about the bounding rectangle, why is this a `QRegion` instead of a `QRect`? I don't currently see a use-case for using the region.